src/FondOfSpryker/Yves/GoogleTagManagerSessionConnector/Expander/SessionDataLayerExpander.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+namespace FondOfSpryker\Yves\GoogleTagManagerSessionConnector\Expander;
+
+use FondOfSpryker\Shared\GoogleTagManagerSessionConnector\GoogleTagManagerSessionConnectorConstants;
+use FondOfSpryker\Yves\GoogleTagManagerSessionConnector\Dependency\GoogleTagManagerSessionConnectorToSessionClientInterface;
+
+class SessionDataLayerExpander implements SessionDataLayerExpanderInterface
```

Comment:
SessionDataLayerExpander -> DataLayerExpander
SessionDataLayerExpanderInterface -> DataLayerExpanderInterface

Reply:
Maybe better move the interface to fond-of-spryker/google-tag-manager-extension?